src/core/utils.js

```diff
@@ -1,8 +1,9 @@
 'use strict';
 
-function getMantissaDecimalString(bignum) {
-  let mantissa = bignum.toPrecision(16)
-    .replace(/\./, '')      // remove decimal point
+// Should only be used directly when the string is from a BigNumber
```

Comment:
You're correct. If the ```mantissa.length``` is greater than 16 should it return it or should it add trailing zeros to a different length (maybe adding a parameter to the function?)?